app/backend/gwells/documents.py

```diff
@@ -161,7 +161,7 @@ def format_object_name(self, object_name: str, document_id: int, resource='well'
 
     def get_documents(self, document_id: int, resource='well', include_private=False):
         """Retrieves a list of available documents for a well or aquifer"""
-
+        logger.info('Get Documents for: document_id:%s, resource: %s, include_private: %s ', document_id, resource, include_private)
```

Comment:
Not that it really matters, but since GWELLS is on Python 3.6 you could also use f-strings which can be easier to read; e.g. `logger.info(f'Get Documents for: document_id:{document_id}...`